src/main/java/uk/gov/ons/ctp/response/casesvc/service/CaseService.java

```diff
@@ -157,7 +159,7 @@ public Case findCaseByIac(final String iac) throws CTPException {
    * @return the cases for the partyId
    */
   public List<Case> findCasesByPartyId(final UUID partyId, boolean iac) {
-    log.debug("Entering findCasesByPartyId");
+    log.info("Entering findCasesByPartyId");
```

Comment:
This log line is just noise, I would just delete it

Reply:
Will turn them down to debug , but it is following the existing pattern of logging on entry

Reply:
It's noise unless you put some context with it, if you log it with the party id then it'd be useful

Reply:
As a general point , I basically followed the existing convention on these log lines on entry as its my first foray into case. Ok I upped some from debug to info( and am now turning them back ). I think its worth agreeing what we want and applying it across the board in the service?

Reply:
I would never consider this useful information in production at debug or info - it's the sort of thing I add to my code while I'm writing it and then get rid of before checking in.